tst/EnergyPlus/api/TestDataTransfer.c

```diff
@@ -109,6 +109,9 @@ void afterZoneTimeStepHandler(EnergyPlusState state)
     printf("Actuated Dew Point temp value is: %8.4f \n", dp_temp);
     Real64 simTime = currentSimTime(state);
     printf("Current Sim Time: %0.2f \n", simTime);
+    const int epwYear = year(state);
+    const int runPeriodYear = calendarYear(state);
+    printf("year: %i, calendarYear: %i\n", epwYear, runPeriodYear);
```

Comment:
Ensure both calls work. This is done for a design day, so runPeriodYear = 0.
But it doesn't crash